ui/app/templates/components/identity/item-alias/alias-metadata.hbs

```diff
@@ -1,17 +1,15 @@
 {{#each-in model.metadata as |key value|}}
-  <div class="box is-sideless is-marginless">
```

Comment:
I was confused about this for a minute - does this still work with the removed columns and is-mobile wrapper? If so we may not need all of the column bits either? Or is that working because info-table-row has some column shared styles?

Reply:
Yep, it was working because of those shared styles. Unfortunately it did change on mobile, so thanks for mentioning that! I meant to check. Fixed now